pkgs/build-support/cc-wrapper/utils.sh

```diff
@@ -25,7 +25,10 @@ mangleVarBool() {
     for infix in "${role_infixes[@]}"; do
         local inputVar="${var/+/${infix}}"
         if [ -v "$inputVar" ]; then
-            let "${outputVar} |= ${!inputVar}"
+            local inputValue="${!inputVar}"
+            # "1" in the end makes `let` return success error code when
+            # expression itself evaluates to zero.
+            let "${outputVar} |= ${inputValue:-0}" "1"
```

Comment:
I've just done `|| true` elsewhere, which I think is a bit easier to read

Reply:
The problem with `|| true` is that we cannot distinguish between a syntax error (`inputValue="00ahaha"`) and legitimate zero.

Reply:
Oooo good point.